front-end/src/screens/Discussions/index.tsx

```diff
@@ -14,6 +15,17 @@ const Discussions = ({ className } : {className?: string}) => {
 		history.push('/post/create');
 	};
 
+	const infoBoxVisible = localStorage.getItem('discussionsInfo');
+	if (infoBoxVisible === null) {
+		localStorage.setItem('discussionsInfo', 'true');
+	}
+	const [infoVisible, setInfoVisible] = useState(infoBoxVisible);
+
+	const handleDismiss = () => {
```

Comment:
same, there're a lot of things we can dismiss. We should be more specific and use e.g `handleInfoBoxDismiss`

Reply:
you can forget about this comment if the `handleDismiss` is inside InfoBox, because it because obvious then.